packages/manager/apps/catalog/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { Params, useMatches } from 'react-router-dom';
+import { OsdsBreadcrumb } from '@ovhcloud/ods-components/breadcrumb/react';
+
+export type BreadcrumbHandleParams = {
+  data: unknown;
+  params: Params<string>;
+};
+
+type BreadcrumbItem = {
+  label: string;
+  href: string;
+};
+
+type MatchHandle = {
+  breadcrumb?: ({ data, params }: BreadcrumbHandleParams) => Promise<string>;
+};
+
+type Match = {
+  handle: MatchHandle;
+  data: unknown;
+  params: Params<string>;
+};
+
+function Breadcrumb(): JSX.Element {
+  const matches = useMatches();
+  const [categoryCrumbs, setCategoryCrumbs] = useState<BreadcrumbItem[]>([]);
+  const [matchCrumbs, setMatchCrumbs] = useState<BreadcrumbItem[]>([]);
+
+  useEffect(() => {
+    const categoryItem = {
+      label: 'Dashboard',
+      href: 'hub',
+    };
+    setCategoryCrumbs(() => [categoryItem]);
+  }, []);
```

Comment:
you could set this default value when you initialize your state instead of initializing it in an useEffect

Reply:
👍